user-service/controller/user-controller.js

```diff
@@ -5,12 +5,24 @@ import {
   ormCheckIfUserExists as _checkIfUserExists,
   ormUpdateUserPassword as _updateUserPassword,
   ormInvalidateJWT as _invalidateJWT,
+  ormFindUserById as _findUserById,
 } from '../model/user-orm.js'
 import { JWT_EXPIRY } from '../constants.js'
 import jwt from 'jsonwebtoken'
 import bcrypt from 'bcrypt'
 import 'dotenv/config'
 
+export const getUser = async (req, res) => {
+  const { userId } = req
+
+  const user = await _findUserById(userId, { password: 0 })

```

Comment:
Wanted to clarify more on this part, what does the projection do? Pinged you on tele!